stdlib/ballerina-builtin/src/main/ballerina/cache/Package.md

```diff
@@ -0,0 +1,21 @@
+## Package Overview
+This package offers a cache that supports time-based eviction, and size-based eviction. This means that entries are removed from the cache once the expiry time has elapsed (i.e., time-based eviction), or when the cache runs out of space (i.e., size-based eviction). Caching is useful when a value takes significant cost or time to compute and retrieve, and it is therefore useful to cache it for future use.
+
+### Sample cache
+The following sample creates a cache that can hold a maximum of 10 entries. An entry of this cache expires one minute after its last access. When the cache is full, 20% of the entries (i.e., 2 entries) will be evicted to make space for new entries.
+
+```ballerina
+import ballerina/cache;
```

Comment:
Isnt it better to add a complete working example with the main function? 

```ballerina
import ballerina/cache;
import ballerina/io;

function main(string... args) {
    // The `evictionFactor` specifies the percentage of items
    // that will be evicted when the cache is full.
    cache:Cache cache = new(capacity = 10, expiryTimeMillis = 60000, evictionFactor = 0.2);

    // Enter a value, and a key to reference that value.
    // This will be a new entry in the cache.
    cache.put("key-a", "value-a");
    // Use the same key to retrieve the value.
    string cachedValue = <string>cache.get("key-a");
    // Print the cached value.
    io:println(cachedValue);
}
```

Reply:
The purpose of the package summery is not to provide a fully working samples, that falls into BBE.

Reply:
Agree.. Looks weird with the import statement. The other option would be to remove the import?

Reply:
See [Ballerina Package Documentation Guidelines](https://docs.google.com/document/d/1wyM5eXGRfITQGXUBbwhMUFhMsgYurJjj_fBOHYfdsE8/edit#heading=h.8edckrk3yir3) doc.

Reply:
@pubudu91 +1 to remove the import statement. It looks odd to me as well.

Reply:
fixed in 5bd05f5